presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -96,18 +100,14 @@ public static Builder builder(String name)
         private final String name;
 
         @SuppressWarnings("resource")
-        private Network network = Network.builder()
-                .createNetworkCmdModifier(createNetworkCmd ->
-                        createNetworkCmd
-                                .withName(PRODUCT_TEST_LAUNCHER_NETWORK)
-                                .withLabels(ImmutableMap.of(PRODUCT_TEST_LAUNCHER_STARTED_LABEL_NAME, PRODUCT_TEST_LAUNCHER_STARTED_LABEL_VALUE)))
-                .build();
+        private final Network network;
 
-        private Map<String, DockerContainer> containers = new HashMap<>();
+        private final Map<String, DockerContainer> containers = new HashMap<>();
 
         public Builder(String name)
         {
             this.name = requireNonNull(name, "name is null");
+            this.network = new ReusableNetwork("ptl-" + name);
```

Comment:
I chose a fixed network name so that we do not produce garbage.
Or, more accurately, even if we produce garbage we will clean it up
the next time.

Reply:
This is not garbage. Docker removes this network after last container using it is killed

Reply:
Are you sure of it? I remember manually cleaning docker networks (back when we were using docker-compose)
Can it depend on how the network was created? Or is my memory wrong?

Reply:
probably Ryuk container deletes unused networks after few seconds?; anyway, I'm not sure about its behavior, so there's also manual removing of the networks during `env down`